### PR TITLE
Updating issue templates

### PR DESCRIPTION
**Pull request by fiorisi**
As per title.

cc @icub-tech-iit/silo-mech

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Request_new_generic_libraries_CAD_part.md [hunk 1]
```diff
@@ -7,24 +7,30 @@ assignees: ''
...
-- [ ] Added assignees
-- [ ] Added label "Libraries-Request" (could be automatic )
-- [ ] Link/datasheet/drawing or other informations needing to create the libraries parts
+### Notes:
 
-without these informations is not possible proceed with the request issue
+
+### Checklist:
+- [ ] Created the Alias following [the standard convention](https://github.com/icub-tech-iit/cad-libraries/wiki/Mechanical-design-guidelines#commercial-components-coding-standard)
+- [ ] Added source files (STEP, IGS, etc)
```
Comment (marcellosavoldi) [inline .github/ISSUE_TEMPLATE/Request_new_generic_libraries_CAD_part.md:28]: Coul be interesting merge the row below:
Added source files (STEP, IGS, etc or\and datasheet, drawing etc )

File: .github/ISSUE_TEMPLATE/Request_new_generic_libraries_CAD_part.md [hunk 2]
```diff
@@ -7,24 +7,30 @@ assignees: ''
...
+- [ ] Added link/datasheet/drawing or other informations needing to create the libraries parts
```
Comment (marcellosavoldi) [inline .github/ISSUE_TEMPLATE/Request_new_generic_libraries_CAD_part.md:29]: see comment above

File: .github/ISSUE_TEMPLATE/Request_new_generic_libraries_CAD_part.md [hunk 3]
```diff
@@ -7,24 +7,30 @@ assignees: ''
...
+- [ ] File imported using the Creo [start part](https://github.com/icub-tech-iit/cad-libraries/wiki/Mechanical-design-guidelines#cad-documentation-standards)
```
Comment (marcellosavoldi) [inline .github/ISSUE_TEMPLATE/Request_new_generic_libraries_CAD_part.md:30]: for the future, could be better convert the file using EDU-Converter

**Review by marcellosavoldi — CHANGES_REQUESTED** [review]
see notes.
for others all ok!

**Comment by fiorisi** [comment]
@marcellosavoldi I've addressed your comments.

**Pull request merged**